pyQuARC/main.py

```diff
@@ -143,34 +145,30 @@ def validate(self):
             metadata_format=self.metadata_format,
             checks_override=self.checks_override,
             rules_override=self.rules_override,
-            messages_override=self.messages_override
+            messages_override=self.messages_override,
         )
 
         if self.concept_ids:
             for concept_id in tqdm(self.concept_ids):
-                downloader = Downloader(concept_id, self.metadata_format, self.version, self.cmr_host)
+                downloader = Downloader(
+                    concept_id, self.metadata_format, self.version, self.cmr_host
+                )
                 if not (content := downloader.download()):
                     self.errors.append(
                         {
                             "concept_id": concept_id,
-                            "errors": [],
-                            "pyquarc_errors": [
-                                {
-                                    "message": "No metadata content found. Please make sure the concept id is correct.",
-                                    "details": f"The request to CMR {self.cmr_host} failed for concept id {concept_id}",
-                                }
-                            ]
+                            "errors": {},
+                            "pyquarc_errors": downloader.errors,
```

Comment:
remove generic error, retrieve the error from downloader - which saves the error from the actual cmr response